routes/webhook.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+const stripe = require('stripe')(
+    'sk_test_51Lb9wkAAgyKcvNJTRqDAxhoN8BH7ke0cYDHUIJW2n4VDJo4py8iq94QscVh518PpJ67FnvLLD9imJlIPuCC7YUkd00HamZx0mC'
```

Comment:
you dont want to expose your api keys in github. look into ways to hide secrets in nodejs https://www.geeksforgeeks.org/where-should-secret-keys-should-be-stored-for-a-node-js-app/

Reply:
Kk good reminder, these are public test keys from stripe that they say are safe for testing but ill put these in an env variable just to be safe